feat/amazon_support_for_adview (#292)
AppLovin/AppLovin-MAX-React-Native: CHANGELOG.md

## Versions

## x.x.x

> * Update support for Amazon bidding with the banner and Mrec native methods.

Reviewer: ```suggestion
* Add support for Amazon bidding for native UI component banners and MRECs.
```
